src/v0/destinations/google_adwords_offline_conversions/transform.js

```diff
@@ -48,6 +49,9 @@ const getConversions = (message, metadata, { Config }, event, conversionType) =>
       filteredCustomerId,
       eventLevelConsentsData,
     );
+    convertedPayload.payload.conversions[0] = updateConversion(
```

Comment:
Are no checks required here on non-empty conversions here ? And why handling only one conversion ? @ItsSudip

Reply:
We don't need to check that. The way config json is written will always create one conversion.

Reply:
then why do we have it as an array ? legacy code ?

Reply:
We need to send this in array format only. Conversions is an array of conversion objects.

Reply:
Then handling in as part of payload preparation itself would have sufficed - https://github.com/rudderlabs/rudder-transformer/blob/develop/src/v0/destinations/google_adwords_offline_conversions/utils.js#L385

Now the transformation logic is split in 2 places. Is updateConversion handles if Array.isArray(products) is false ?

Reply:
No that is not required. That is already being handled before the `updateConversion` function call.